onvm/onvm_mgr/onvm_stats.h

```diff
@@ -57,6 +57,15 @@
 #define ONVM_STR_STATS_STDERR "stderr"
 #define ONVM_STR_STATS_WEB "web"
 
+extern const char *NF_MSG[3];
+#define ONVM_STATS_MSG "\nNF TAG         IID / SID / CORE    rx_pps  /  tx_pps        rx_drop  /  tx_drop"\
```

Comment:
Lets try 
```
#define ONVM_STATS_MSG \
    "\nNF TAG         IID / SID / CORE    rx_pps  /  tx_pps        rx_drop  /  tx_drop         out   /    tonf     /   drop\n"\
...
```
This way its cleaner to read. Also I feel like its fine if it goes over the char limit for these definitions

Reply:
Ok sounds good, and line limits are going to be the only lint errors in this pr which is fine I guess